js/src/tooltip.js

```diff
@@ -340,7 +340,9 @@ class Tooltip {
       this._cleanTipClass()
       this.element.removeAttribute('aria-describedby')
       EventHandler.trigger(this.element, this.constructor.Event.HIDDEN)
-      this._popper.destroy()
+      if (this._popper !== null) {
```

Comment:
I would add that check at the beginning of the hide function, something like:
```js
if (!this._popper) {
  return
}

Reply:
I wasn't really sure if the code before the destroy was still relevant, but i can move it if you think that's better.

Reply:
A tooltip cannot be shown without Popper.js

Reply:
True, good catch and suggestion 👍